src/ngx_http_lua_kong_module.c

```diff
@@ -20,6 +20,7 @@
 
 static ngx_int_t ngx_http_lua_kong_init(ngx_conf_t *cf);
 static void* ngx_http_lua_kong_create_loc_conf(ngx_conf_t* cf);
+static char* ngx_http_lua_kong_merge_loc_conf(ngx_conf_t *cf, void *parent, void *child);
```

Comment:
I think we need not to enable `merge` feature. Defining in `location` is enough.

Reply:
I think being able to define this directive on the http/server block would be an added value, otherwise we would only be able to provide a correctly formatted error log after the rewrite phase has completed (in location), am I right? And for that to work we'd need the `merge`. At the same time being able to set this in the location block provides added flexibility in case different formats are needed in different locations so I think we should leave all the scopes available. Let me know if you think this makes sense.

Reply:
Merge should be implemented so that it works with the Nginx config semantics.